### DISCUSSION TITLE
Anyone having an issue with meeting display today?

### DISCUSSION THREAD

**Discussion by lbardiv** [code4recovery/12-step-meeting-list#1179] [General]
Was notified today from a user that not all meetings are displaying. I'm also seeing that it's not working.

**Comment by joshreisner** [comment]
hi @lbardiv what's your website URL?

**Reply by joshreisner** [reply]
@lbardiv thanks i'll get that fixed, sorry about that

**Reply by joshreisner** [reply]
@lbardiv thanks if i were more awesome i wouldn't have caused the problem 😅

should be fixed now if you refresh the page. might need to do a "hard refresh" which on most computers if shift command/ctrl r - on mobile quitting and reopening

**Comment by lbardiv** [comment]
Ndiaa.org

Thanks for the quick response!!

**Comment by lbardiv** [comment]
You're awesome!

**Comment by lbardiv** [comment]
Lol thank you so much!!
